pom.xml

```diff
@@ -108,6 +108,11 @@
             <artifactId>maven-model</artifactId>
             <version>3.3.9</version>
         </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
```

Comment:
Maven Shade Plugin doesn't work due to this Maven HPI Plugin bug: https://issues.jenkins-ci.org/browse/JENKINS-31133 ...